dashboard/test/models/user_test.rb

```diff
@@ -61,12 +61,6 @@ class UserTest < ActiveSupport::TestCase
     assert user.errors[:email].length == 1
   end
 
-  test "cannot create young user with invalid email" do
```

Comment:
Duplicated by previous test as we no longer treat young students and old students differently with respect to email.